Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL Description="Tyk Pump docker image" Vendor="Tyk" Version=$TYKVERSION
 RUN apt-get update \
  && apt-get upgrade -y \
  && apt-get install -y --no-install-recommends \
-            curl ca-certificates apt-transport-https \
+            curl ca-certificates apt-transport-https gnupg \
```

Comment:
Should probably uninstall `gnupg` after it's not needed anymore to save a bit on the image size.

Reply:
Why is it not needed anymore? If I remove `gnupg` I got an error:

```
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation

Reply:
I mean, if it's only used to install the package, it can then be removed afterwards so that unnecessary packages aren't baked into the final image.